invisible_cities/cities/hypathia.py

```diff
@@ -0,0 +1,160 @@
+"""
+-----------------------------------------------------------------------
+                                 Hypathia
+-----------------------------------------------------------------------
+
+From ancient Greek ‘Υπατια: highest, supreme.
+
+This city reads true waveforms from detsim and compute pmaps from them
+without simulating the electronics. This includes:
+    - Rebin 1-ns waveforms to 25-ns waveforms to match those produced
+      by the detector.
+    - Produce a PMT-summed waveform.
+    - Apply a threshold to the PMT-summed waveform.
+    - Find pulses in the PMT-summed waveform.
+    - Match the time window of the PMT pulse with those in the SiPMs.
+    - Build the PMap object.
+"""
+import numpy  as np
+import tables as tb
+
+from .. types.ic_types import minmax
+from .. database       import load_db
+
+from .. reco                  import tbl_functions        as tbl
+from .. reco                  import  peak_functions      as pkf
```

Comment:
Minor misalignment of `peak_functions`

Reply:
Fixed in the last commit with some other missalignments in the imports.